test/integration/propControls/index.spec.ts

```diff
@@ -40,35 +24,28 @@ test('can control component prop values in properties control panel', async ({
   const firstInputLocator = canvasInputLocator.first();
   await clickCenter(page, firstInputLocator);
 
-  await editorModel.componentEditor.waitFor();
+  await editorModel.componentEditor.waitFor({ state: 'visible' });
+
+  const labelControlInput = editorModel.componentEditor.getByLabel('label', { exact: true });
```

Comment:
i guess i hadn't found this selector that gets the input for a label? seems pretty useful